src/dom/dom.ts

```diff
@@ -518,7 +518,7 @@ export class dom {
      * so memory can be released.
      *
      * ```
-     * const imageSource = '/public/image/cat.jpg';
+     * const imageSource = './public/image/cat.jpg';
```

Comment:
might want to revert these within jsdocs since a regular might be confused between relative path and absolute.